### PR TITLE
Finishing desktop version

### PR DESCRIPTION
**Pull request by kazaneza**
- Added desktop version
- Used media queries
- Solved linter errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,39 +1,348 @@
...
+          <li class="desktop-link"><a href="#">contact</a></li>
+        </ul>
+        <a href="#" class="mobile-nav">
+          <img
+            class="navImage"
+            src="./Images/ic_menu.svg"
+            alt="Navigation menu"
+          />
+        </a>
+      </nav>
```
Comment (iambenkis) [inline index.html:36]: - [ ] Good job :+1:  implementing this website, However your header should stick to the design as it is a crucial part of the project requirement. A design stick to another one if argument such as `font`, `color` `margin` and so on match :100:  Kindly make sure you correct that

Your design

Screenshot from 2022-12-10 18-19-23

Figma design

Screenshot from 2022-12-10 18-19-54

File: index.html [hunk 2]
```diff
@@ -1,39 +1,348 @@
...
+          <li>
+            <a href=""><img src="./images/Group-2.png" alt="" /></a>
+          </li>
+          <li>
+            <a href=""><img src="./images/Vector-1.png" alt="" /></a>
+          </li>
+          <li>
+            <a href=""><img src="./images/Vector.png" alt="" /></a>
+          </li>
+        </ul>
```
Comment (iambenkis) [inline index.html:69]: - [ ] Kindly make sure the `space` between these elements match with the `space` in the Figma design.

Your Design
Screenshot from 2022-12-10 18-12-50

Figma Design

Screenshot from 2022-12-10 18-27-56

File: index.html [hunk 3]
```diff
@@ -1,39 +1,348 @@
...
+            <ul class="projectLanguages">
+              <li><img src="./images/Tag (4).png" alt="" /></li>
+              <li><img src="./images/Tag (5).png" alt="" /></li>
+              <li><img src="./images/Tag (6).png" alt="" /></li>
+            </ul>
+          </div>
+          <div>
+            <button>See Project</button>
+          </div>
+        </div>
```
Comment (iambenkis) [inline index.html:171]: - [ ] Good job implementing this `project section` :clap: , However there is something you need to change to meet the requirements. :100: . Note that the `see Project` button is not visible initially, It's when the mouse is over that you can see it. Kindly change that :smile: 

Your Design

Screenshot from 2022-12-10 18-09-58 (1)


Figma Design
Screenshot from 2022-12-10 18-09-24 (1)

File: index.html [hunk 4]
```diff
@@ -1,39 +1,348 @@
...
+              type="text"
+              maxlength="30"
+              required
+              placeholder="Full names"
+            />
+            <input
+              class="input-field"
+              type="email"
+              required
+              placeholder="email address"
```
Comment (iambenkis) [inline index.html:311]: - [ ] Kindly make sure the contact section stick to the design template .

Your Design
Screenshot from 2022-12-10 18-05-51


Figma Design

Screenshot from 2022-12-10 18-14-55

File: index.html [hunk 5]
```diff
@@ -1,39 +1,348 @@
...
+          </li>
+          <li>
+            <a href=""><img src="./images/vector2.png" alt="" /></a>
+          </li>
+          <li>
+            <a href=""><img src="./images/vector3.png" alt="" /></a>
+          </li>
+          <li>
+            <a href=""><img src="./images/vector4.png" alt="" /></a>
+          </li>
```
Comment (iambenkis) [inline index.html:343]: - [ ] Kindly make sure the `space` between these elements match with the `space` in the Figma design.

Your Design
Screenshot from 2022-12-10 18-06-26 (1)


Figma Design

Screenshot from 2022-12-10 18-48-41

**Review by iambenkis — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @kazaneza  👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project, but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:



 ## Required Changes ♻️

- [ ] You should stick to the design as much as possible (e.g. font, colours, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). :+1: 

Check the comments under the review

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me `@iambenkis` in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Abenezer-Tilahun) [reply index.html]: Fixed.

File: index.html (hunk 2 above)
Comment (Abenezer-Tilahun) [reply index.html:72]: Fixed.

File: index.html (hunk 4 above)
Comment (Abenezer-Tilahun) [reply index.html:314]: Fixed.

File: index.html (hunk 5 above)
Comment (Abenezer-Tilahun) [reply index.html:346]: Fixed.

File: index.html (hunk 3 above)
Comment (Abenezer-Tilahun) [inline index.html:174]: - [ Optional ] As requested by the previous reviewer kindly implemented the requested changes , You must replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your Project section Look likes this : 

Capture

The Figma design Look likes this : 

Capture

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @kazaneza 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct GitHub-Flow.
✔️ Professional README  
✔️ Good Commit Messages


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**